src/frontend/api/søknad.ts

```diff
@@ -10,17 +9,7 @@ export const registrerSøknad = (
     aktivBehandling: IBehandling | undefined,
     innloggetSaksbehandler?: ISaksbehandler
 ): Promise<Ressurs<IFagsak>> => {
-    return axiosRequest(
-        {
-            method: 'POST',
-            data: {
-                ...søknad,
-                barnaMedOpplysninger: søknad.barnaMedOpplysninger.filter(
-                    (it: IPartMedOpplysninger) => it.checked
-                ),
-            },
-            url: `/familie-ba-sak/api/behandlinger/${aktivBehandling?.behandlingId}/registrere-søknad-og-hent-persongrunnlag`,
-        },
+    return ,
         innloggetSaksbehandler
```

Comment:
`ISaksbehandler | undefined er not assignlable to type 'Promise<Resurs<IFagsak>>'`, og const'en registrerSøknad brukes heller ikke :)